tests/stress_tests/.automation/unittests/nightly_configs/desktop_test_config.xml

```diff
@@ -1,6 +1,10 @@
 <attributes>
     <processes>
         <value>1</value>
+        <value>2</value>
+        <value>4</value>
+        <value>8</value>
+        <value>16</value>
```

Comment:
@alexander-shchepetov @just-sparta what's the reason to add such number of cases? I would prefer to add only 1 case (and 2 in weekly) with, e.g. 2 processes, and increase number of models. Note that need to keep tracking execution time (with the change tests will consume x5 time)

Reply:
Done